packages/babylon/src/tokenizer/index.js

```diff
@@ -114,7 +117,7 @@ function codePointToString(code: number): string {
   } else {
     return String.fromCharCode(
       ((code - 0x10000) >> 10) + 0xd800,
-      ((code - 0x10000) & 1023) + 0xdc00,
+      ((code - 0x10000) & charCodes.lowercaseF) + 0xdc00,
```

Comment:
f is 102, not 1023

Reply:
Yeah that's due to my search-and-replace. I ended up with `charCodes.lowercaseF3` at this place. Thanks, i'll update it.